Types error when import from react-native

### Description

There is an issue with the `View` type exported from the react-native module. 
```
error TS2693: 'View' only refers to a type, but is being used as a value here.
```
The code is not different from the template application.

### Version

0.71.10

### Output of `npx react-native info`

System:
    OS: macOS 13.4
    CPU: (12) arm64 Apple M2 Pro
    Memory: 102.86 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 19.9.0 - /opt/homebrew/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 9.6.4 - /opt/homebrew/bin/npm
    Watchman: 2023.04.10.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.12.0 - /Users/tikhonov/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    Android SDK:
      API Levels: 28, 31, 32, 33, 33
      Build Tools: 30.0.2, 30.0.3, 31.0.0, 33.0.0, 33.0.2
      System Images: android-28 | Google ARM64-V8a Play ARM 64 v8a, android-30 | Google TV Intel x86 Atom, android-30 | Google APIs ARM 64 v8a, android-31 | Android TV ARM 64 v8a, android-33 | Android TV ARM 64 v8a, android-33 | Android TV Intel x86 Atom, android-33 | Google TV ARM 64 v8a, android-33 | Google TV Intel x86 Atom, android-33 | Google APIs ARM 64 v8a, android-33 | Google APIs Intel x86_64 Atom, android-33 | Google Play ARM 64 v8a, android-33 | Google Play Intel x86_64 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2022.1 AI-221.6008.13.2211.9619390
    Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.18 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0 
    react-native: Not Found
    react-native-macos: Not Found
    react-native-tvos:  0.71.10-0 
  npmGlobalPackages:
    *react-native*: Not Found


### Steps to reproduce

1. Create a template app with the `react-native init TestApp --template=react-native-template-typescript-tv` command
2. Run `yarn tsc --noEmit ` 

### Snack, code example, screenshot, or link to a repository

```
import React from 'react'
import {Text, View} from 'react-native'

function App(): JSX.Element {
  return (
    <View>
      <Text>Test</Text>
    </View>
  )
}

export default App
```

--- a/types/public/ReactNativeTVTypes.d.ts
+++ b/types/public/ReactNativeTVTypes.d.ts
@@ -31,7 +31,7 @@ declare module 'react-native' {
   nextFocusUp?: number,
   }
 
-  interface View {
+  export interface NativeMethods {
     requestTVFocus(): void;
   }
 

